tox.ini

```diff
@@ -9,22 +11,29 @@ deps =
     readthedocs-sphinx-ext
     pytest
     sphinxcontrib-httpdomain
-    sphinx16: Sphinx < 1.7
-    sphinx17: Sphinx < 1.8
-    sphinx18: Sphinx < 1.9
-    sphinx20: Sphinx < 2.1
-    sphinx21: Sphinx < 2.2
-    sphinx22: Sphinx < 2.3
-    sphinx23: Sphinx < 2.4
-    sphinx24: Sphinx < 2.5
-    sphinx30: Sphinx < 3.1
-    sphinx31: Sphinx < 3.2
-    sphinx32: Sphinx < 3.3
-    sphinx33: Sphinx < 3.4
-    sphinx34: Sphinx < 3.5
-    sphinx35: Sphinx < 3.6
-    sphinx40: Sphinx < 4.1
-    sphinx41: Sphinx < 4.2
+    sphinx16: Sphinx>=1.6,<1.7
+    sphinx17: Sphinx>=1.7,<1.8
+    sphinx18: Sphinx>=1.8,<1.9
+    sphinx20: Sphinx>=2.0,<2.1
+    sphinx21: Sphinx>=2.1,<2.2
+    sphinx22: Sphinx>=2.2,<2.3
+    sphinx23: Sphinx>=2.3,<2.4
+    sphinx24: Sphinx>=2.4,<2.5
+    sphinx30: Sphinx>=3.0,<3.1
+    sphinx31: Sphinx>=3.1,<3.2
+    sphinx32: Sphinx>=3.2,<3.3
+    sphinx33: Sphinx>=3.3,<3.4
+    sphinx34: Sphinx>=3.4,<3.5
+    sphinx35: Sphinx>=3.5,<3.6
+    sphinx40: Sphinx>=4.0,<4.1
+    sphinx41: Sphinx>=4.1,<4.2
+    sphinx42: Sphinx>=4.2,<4.3
+    sphinx43: Sphinx>=4.3,<4.4
+    sphinx44: Sphinx>=4.4,<4.5
+    sphinx45: Sphinx>=4.5,<4.6
+    sphinx50: Sphinx>=5.0,<5.1
+    sphinx51: Sphinx>=5.1,<5.2
+    sphinx{16,17,18,20,21,22,23,24,30,31,32,33,34,35,40,41,42}: docutils<0.18
```

Comment:
The latter line here is not important for the current setup, but it will be important once we bump sphinx_rtd_themes docutils dependency to `<0.18` in #1304